src/styles/01-atoms/text/headings/_headings.scss

```diff
@@ -6,7 +6,7 @@
 @mixin heading-xl($color: $black) {
   color: #{$color};
   font-size: 2.25rem;
-  font-weight: normal;
+  font-weight: lighter;
```

Comment:
@helenvholmes Every time I see H1 (even in screenshots from reno) it seems like it's light -- is this an ok change to make?

Reply:
You are completely right. I'm going to push up some changes I've made in Reno to this file.

I got there by following the weights set in [this library](https://github.com/jonathantneal/system-font-css/blob/gh-pages/system-font.css)—I assume `lighter` works too, but I felt this was easier for mapping back later... this would probably be good documentation to write down at some point.